concent_api/conductor/models.py

```diff
@@ -0,0 +1,50 @@
+from django.db.models       import CharField
+from django.db.models       import DateTimeField
+from django.db.models       import ForeignKey
+from django.db.models       import Model
+from django.utils           import timezone
+
+from core.constants         import MESSAGE_TASK_ID_MAX_LENGTH
+
+
+class VerificationRequest(Model):
+
+    # Fields from ComputeTaskDef that we actually use.
+    subtask_id              = CharField(max_length = MESSAGE_TASK_ID_MAX_LENGTH, null = True, blank = True)
+    src_code                = CharField(max_length = 255, null = True, blank = True)
+    extra_data              = CharField(max_length = 255, null = True, blank = True)
+    short_description       = CharField(max_length = 255, null = True, blank = True)
+    working_directory       = CharField(max_length = 255, null = True, blank = True)
+    performance             = CharField(max_length = 255, null = True, blank = True)
+    docker_images           = CharField(max_length = 255, null = True, blank = True)
+
+    # Indicates when Conductor has received the request.
+    created_at              = DateTimeField(default=timezone.now)
+
+
+class UploadRequest(Model):
+    """
+    Existence of this object indicates that Concent is expecting a client to upload a specific file.
+    """
+
+    # Foreign key to VerificationRequest. Can't be NULL.
+    verification_request = ForeignKey(VerificationRequest, related_name='upload_requests')
+
+    # Relative path to the same directory that paths listed in FileTransferTokens are relative to. Must be unique.
+    path                 = CharField(max_length = 255, unique = True)
```

Comment:
Please define a constant for path length.

And add an assert after the definition that states that the maximum path length must be greater than the maximum `subtask_id` length.

Reply:
:+1: